src/app/modules/auth/containers/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,3 @@
+.p-float-label {
+  margin: 20px 0;
```

Comment:
you can do this with css classes no need to have it here
look up how to do it in primeflex